src/current/_includes/v25.1/cdc/privilege-model.md

```diff
@@ -1,9 +1,29 @@
+We recommend transitioning users that need to view and manage running changefeed jobs to [roles]({% link {{ page.version.version }}/create-role.md %}) that own the [jobs]({% link {{ page.version.version }}/show-jobs.md %}) or [granting]({% link {{ page.version.version }}/grant.md %}) them the `VIEWJOB` or `CONTROLJOB` privilege. For more details, refer to [View and manage changefeed jobs](#view-and-manage-changefeed-jobs).
 {{site.data.alerts.end}}
 
+### Privilege model
+
+The following summarizes the operations users can run depending on whether the assigned privileges are at the job or table level:
+
+Granted privileges | Usage
+-------------------+-------
+`CHANGEFEED` | Create changefeeds on tables. For details, refer to [`CHANGEFEED` privilege](#changefeed-privilege).<br>**Deprecated**: View and manage changefeed jobs on tables.
+`CHANGEFEED` + [`USAGE`]({% link {{ page.version.version }}/create-external-connection.md %}#required-privileges) on external connection | Create changefeeds on tables to an external connection URI. For details, refer to [`CHANGEFEED` privilege](#changefeed-privilege).<br>**Deprecated**: View and manage changefeed jobs on tables.<br><br>**Note:** If you need to manage access to changefeed sink URIs, set the `changefeed.permissions.require_external_connection_sink.enabled=true` cluster setting. This will mean that users with these privileges can **only** create changefeeds on external connections.
+Job ownership | [View]({% link {{ page.version.version }}/show-jobs.md %}#show-changefeed-jobs) and manage changefeed jobs ([pause]({% link {{ page.version.version }}/pause-job.md %}), [resume]({% link {{ page.version.version }}/resume-job.md %}), and [cancel]({% link {{ page.version.version }}/cancel-job.md %})). For details, refer to [View and manage changefeed jobs](#view-and-manage-changefeed-jobs).
+`CONTROLJOB` | Manage changefeed jobs ([pause]({% link {{ page.version.version }}/pause-job.md %}), [resume]({% link {{ page.version.version }}/resume-job.md %}), and [cancel]({% link {{ page.version.version }}/cancel-job.md %})). For details, refer to [View and manage changefeed jobs](#view-and-manage-changefeed-jobs).
+`VIEWJOB` | [View]({% link {{ page.version.version }}/show-jobs.md %}#show-changefeed-jobs) changefeed jobs. For details, refer to [View and manage changefeed jobs](#view-and-manage-changefeed-jobs).
+`SELECT` | Create a sinkless changefeed that emits messages to a SQL client.
+**Deprecated** `CONTROLCHANGEFEED` role option + `SELECT` | Create changefeeds on tables.
+
+{{site.data.alerts.callout_info}}
+Users with the `CONTROLCHANGEFEED` role option must have `SELECT` on each table, even if they are also granted the `CHANGEFEED` privilege. The `CONTROLCHANGEFEED` role option will be deprecated in a future release.
```

Comment:
According to this page [1] the `CONTROLCHANGEFEED` role option was deprecated in v23.1 ?  IIUC that means either this is wrong or that is wrong (?)

 Also that role option's docs at [1] say to use the (now deprecated) `CHANGEFEED` privilege, so it feels like that should be updated too, although if you update the `CHANGEFEED` privilege docs at [2], the link from `CONTROLCHANGEFEED` will at least eventually get me to the right info after another click 

[1]: https://deploy-preview-19290--cockroachdb-docs.netlify.app/docs/v25.1/create-role#role-options
[2]: https://deploy-preview-19290--cockroachdb-docs.netlify.app/docs/v25.1/security-reference/authorization#supported-privileges

Reply:
So the `CHANGEFEED` privilege is not deprecated in its entirety, only the view and manage changefeed functionality. I'll take another look at the initial note as that may not be clear enough.